README.md

```diff
@@ -1,47 +1,38 @@
 # Asgardio Tomcat SAML Agent
 
-## Table of Contents
-- [Trying out the sample](#trying-out-the-sample)
-  * [Prerequisites](#prerequisites)
-  * [Running the sample](#running-the-sample)
-    + [Configuring Identity Server](#configuring-identity-server)
-    + [Configuring the sample](#configuring-the-sample)
+Asgardio Tomcat SAML Agent enables you to add SAML based login, logout to your Apache Tomcat web apps in simple manner.
+
+- [Getting Started](#getting-started)
 - [How it works](#how-it-works)
-  * [Classify secure resources, unsecured resources](#classify-secure-resources--unsecured-resources)
-  * [Trigger authentication](#trigger-authentication)
-  * [Retrieve user attributes](#retrieve-user-attributes)
-  * [Trigger logout](#trigger-logout)
-- [Integrating SAML into your Java application](#integrating-saml-into-your-java-application)
-  * [Getting started](#getting-started)
-    + [Prerequisites](#prerequisites-1)
-  * [Configuring the sample](#configuring-the-sample)
-  * [Enable login](#enable-login)
-  * [Enable logout](#enable-logout)
-  * [Retrieving user attributes](#retrieving-user-attributes)
+- [Integrating SAML into your Java Source Project](#integrating-saml-into-your-java-application)
 - [Integrating SAML into your Existing Webapp](./docs/integrating_with_existing_webapp.md/#Integrating_SAML_into_your_existing_Webapp)
-    * [Getting started](./docs/integrating_with_existing_webapp.md/#getting-started)
-      + [Prerequisites](./docs/integrating_with_existing_webapp.md/#prerequisites-1)
-    * [Configuring the sample](./docs/integrating_with_existing_webapp.md/#configuring-the-sample)
-    * [Enable login](./docs/integrating_with_existing_webapp.md/#enable-login)
-    * [Enable logout](./docs/integrating_with_existing_webapp.md/#enable-logout)
-    * [Retrieving user attributes](./docs/integrating_with_existing_webapp.md/#retrieving-user-attributes)
 - [Building from the source](#building-from-the-source)
 - [Contributing](#contributing)
   * [Reporting issues](#reporting-issues)
 - [License](#license)
 
-Throughout this guide we will refer to the sample-app installation directory as APP_HOME.
 
-## Trying out the sample
+## Getting started
+
+You can experience the capabilities of Asgardio Tomcat SAML Agent by following this small guide which contains main sections listed below.
+
+  * [Prerequisites](#prerequisites)
+  * [Configuring Identity Server](#configuring-identity-server)
```

Comment:
Shouldn't the "Configuring the sample" and "Configuring the Identity Server"  swap in order?

Reply:
why?

Reply:
If there are not dependency, better to have it as this order since this guide is mainly about the sample app. So having that one first would be better IMO.

Reply:
You meant the ToC order and guide order are different.. Fixed that with https://github.com/asgardio/asgardio-tomcat-saml-agent/pull/8/commits/b161243b28fd6409693510d8b520e02066744e6e